nfabackend/pom.xml

```diff
@@ -23,8 +23,8 @@
                 <version>2.5.1</version>
                 <inherited>true</inherited>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>9</source>
```

Comment:
Do I have to install/upgrade something for this change to work? or Does IDE takes care of dependencies automatically?
Could you please write down the steps that you did in your local machine for this change to work.?

Reply:
I agree. It would be nice if you wrote in the description, what we have to do in order to benefit from this merge.
It appears to be in working condition though.

Reply:
i did nothing, only changed manually 1.7 first to 1.9, but it didnot work. After that i found in settings not 1.9, but 9 need to be used and simply wrote that down. We did not you till now any special Java 9 feature. So it work whis that changes